remote-config.jsonc

```
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
```

I’m adding scope creep, but these might as well follow the same format and be complete sentences with punctuation: 

```suggestion
                    "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests."
```